venter/models.py

```diff
@@ -20,10 +20,24 @@ class Meta:
     def __str__(self):
         return self.tag_uri
 
+
+class Authorities(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
+    name = models.CharField(max_length=30, blank=True, null=True)
+    email = models.EmailField(max_length=50, blank=True, null=True, unique=True)
+
+    class Meta:
+        verbose_name = 'Authority Email'
+        verbose_name_plural = 'Authority Emails'
+
+    def __str__(self):
+        return self.email
+
+
 class Complaints(models.Model):
     id = models.UUIDField(primary_key=True, default=uuid4, editable=False)
     created_by = models.ForeignKey(
-        'users.UserProfile', on_delete=models.CASCADE, related_name='created_by')
+        'users.UserProfile', on_delete=models.CASCADE, related_name='created_by', null=True)
```

Comment:
Ah yes, that was a silly mistake. I was having issues with new complaints having a 'null' in the created_by field when i tried accessing my admin panel and adding complaints there. I think this was happening because of the read-only property given to the 'created_by' field in the admin code here:
`
class ComplaintModelAdmin(admin.ModelAdmin):
    **readonly_fields = ['created_by']**
    list_display = ['report_date', 'status', 'authority_email']
    list_editable = ['status', 'authority_email']
    list_filter = ['status']
`

Thus, I tested my new complaints while commenting it out, but I kept it as it was for the commit.